packages/electron-builder-util/src/util.ts

```diff
@@ -163,10 +163,14 @@ export function getTempName(prefix?: string | null | undefined): string {
   return `${prefix == null ? "" : `${prefix}-`}${tempDirPrefix}-${(tmpDirCounter++).toString(16)}`
 }
 
-export function isEmptyOrSpaces(s: string | null | undefined) {
+export function isEmptyOrSpaces(s: string | null | undefined): s is "" | null | undefined {
   return s == null || s.trim().length === 0
```

Comment:
`:arg is type` does not mean the return type. It is type guards (https://www.typescriptlang.org/docs/handbook/advanced-types.html)

It mean that if isEmptyOrSpaces return true, then `s` should be `''` or `null` or `undefined`.
Otherwise, `s` should be: 

`string | null | undefined` - `'' | null | undefined` = `string`

```
let str: string | null
if (!isEmptyOrSpaces(str)) { 
  str.slice // str is string only 
}
```